app/assets/javascripts/oxalis/model/binary/bucket_picker_strategies/oblique_bucket_picker.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import PriorityQueue from "js-priority-queue";
+import { M4x4, V3 } from "libs/mjs";
+import { getPosition } from "oxalis/model/accessors/flycam_accessor";
+import {
+  zoomedAddressToAnotherZoomStep,
+  globalPositionToBucketPosition,
+} from "oxalis/model/helpers/position_converter";
+import type { Vector3, Vector4 } from "oxalis/constants";
+import traverse from "oxalis/model/binary/bucket_traversals";
+import _ from "lodash";
+import type { Matrix4x4 } from "libs/mjs";
+import Binary from "oxalis/model/binary";
+import Store from "oxalis/store";
+import { chunk2 } from "oxalis/model/helpers/chunk";
+
+export default function determineBucketsForOblique(
+  binary: Binary,
+  bucketQueue: PriorityQueue,
+  matrix: Matrix4x4,
+  logZoomStep: number,
+  fallbackZoomStep: number,
+  isFallbackAvailable: boolean,
+): void {
+  const queryMatrix = M4x4.scale1(1, matrix);
+
+  const enlargementFactor = 1.1;
```

Comment:
It's not really necessary in the sense of rendering a static plane. However, it improves movement within the data, because buckets adjacent to the current viewport are already there. Some sort of "prefetching" if you will. But, I'll add a comment to the code :)